jme3-core/src/main/java/com/jme3/renderer/Camera.java

```diff
@@ -485,7 +483,7 @@ public void resize(int width, int height, boolean fixAspect) {
     }
 
     /**
-     * <code>getFrustumBottom</code> returns the value of the bottom frustum
+     * Returns the value of the bottom frustum
```

Comment:
Completely aside from the format/styling concerns, I question the value of this, and other "Trivial" Javadoc comments in this file. It adds no details that cannot be gleaned from the method signature, and therefore does not add value above what the javadoc tool would create on it's own. Why does it exist?

Reply:
The style guide says that javadoc is optional for "simple, obvious" methods like getFoo, in cases where there really and truly is nothing else worthwhile to say but "Returns the foo".

Reply:
This PR show minimal changes. Deleting the entire javadoc seemed like a bigger change than deleting one word.

I agree that the current javadoc isn't valuable. Honestly, I haven't studied the frustum math to figure out what "the value of the bottom frustum" really is. Is it the y/z slope, the y-coordinate at the near plane, or the y-coordinate at the far plane? So I believe `getFrustumBottom()` needs *some* javadoc, but I don't yet know enough to write it myself. Writing javadoc for someone else's code can be very frustrating!

Reply:
Good point, though in this case the code literally passes through the value of a field.

I hear you on documenting other people's code. In the case of the frustrum stuff here, I'd think that the javadoc should go on the field declarations, as they are `protected` and could, in theory, be overridden by extension code.

Reply:
In theory, yes.

Reply:
Ok, there's a proposed change (to the style guide) that would remove the option of omitting javadoc for "simple, obvious" methods. Discussion of this proposed change is at #1098.